internal/annotations/annotations.go

```diff
@@ -155,6 +155,36 @@ func ExtractKongPluginsFromAnnotations(anns map[string]string) []string {
 	return kongPluginCRs
 }
 
+type NamespacedKongPlugin struct {
+	Namespace string
+	Name      string
+}
```

Comment:
This could potentially be a type alias for https://pkg.go.dev/k8s.io/apimachinery/pkg/types#NamespacedName.